auth-server/test/resolvers/mutation/addressLinkStart.spec.ts

```diff
@@ -59,6 +60,34 @@ describe('addressLinkStart mutation', () => {
 		expect(res.message).to.equal(messages.ADDRESS_LINKING_STARTED);
 	});
 
+	it('should not be able to start linking address if it already exists in db', async () => {
+		const network = NetworkEnum.KUSAMA;
+		const address = 'FoQJpPyadYccjavVdTWxpxU7rUEaYhfLCPwXgkfD6Zat9QP'; // Bob
```

Comment:
idem here, we delete the user but not the link in the DB

Reply:
done